px-typeahead.html

```diff
@@ -115,6 +115,15 @@
         type: String,
         value: ''
       },
+      /**
+       * Headers for pretech request
+       */
+      prefetchHeaders: {
+        type: Object,
+        defaultValue: {
```

Comment:
`defaultValue` is a demo prop. You want `value`.

Also, fun gotcha, it needs to be a function that returns an object otherwise it'll leak into other instances of the component eg:
```javascript
value: function() { return {'X-Requested-With': 'XMLHttpRequest'} }
```